Add doc of ports name and description (#2129)
gitpod-io/website: gitpod/docs/references/gitpod-yml.md

```yaml
ports:
  - name: Website
    port: 3000
    onOpen: open-preview
  - name: VNC
    description: full GUI Virtual Desktop
    port: 6080
    onOpen: open-browser
  - name: Server
    port: 10000
    onOpen: ignore
```

### `ports[n].name`

> Define a name for the port, also shown on the vscode sidebar remote explorer.

Reviewer: **Issue:** We should mention CLI users and JetBrains, not only VS Code.

> Define a name for the port, also shown on the VS Code Browser' sidebar `Remote Explorer` and `gp` cli. [More detail](/docs/config-ports#specifying-port-names--descriptions)

Reviewer 2: ```suggestion
Define a name for the port, also shown in the `Remote Explorer` under the VS Code Browser sidebar and the `gp` CLI. [More detail](/docs/config-ports#specifying-port-names--descriptions)
```

<div class="overflow-x-auto">

| Type     | Default   |
| -------- | --------- |
| `string` | `<empty>` |

</div>

### `ports[n].description`

> Define description of the port, also shown as tooltip of port on vscode sidebar remote explorer (hover can show tooltip).

Reviewer 2: ```suggestion
Adds a description to the port, also shown as a tooltip (on hover) of the port in the `Remote explorer` in VS Code's side bar.
```

Author: Thanks ❤️ @filiptronicek

> Adds a description to the port, also shown as a tooltip (on hover) of the port in the `Remote explorer` in VS Code's side bar.

Reviewer: **Issue:** We should mention CLI users and JetBrains, not only VS Code.

Author: There is no cli (gp ports list) before this PR create 😂, I will update this PR later

> Adds a description to the port, will shown as `gp ports list` command `NAME & DESCRIPTION` field and a tooltip (when hover) of the port in the `Remote explorer` in VS Code Browser' sidebar.

Reviewer 2: ```suggestion
Adds a description to the port, will shown as a column in the output of `gp ports list` and as a tooltip (on hover) of the port in the `Remote explorer` in VS Code Browser sidebar.
```

I think just referring to it as a column should be easier to read.
